features/overscroll-behavior.yml

```
@@ -0,0 +1,29 @@
+name: overscroll-behavior
+description: The `overscroll-behavior` CSS property allows disabling default scrolling behaviors when the edges of a scrolling area are reached.
```

Since there's another comment on this that needs action…

https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#allows

```suggestion
description: The `overscroll-behavior` CSS property disables default scrolling behaviors when the edges of a scrolling area are reached.
```